### PR TITLE
Add lvm2 package to use `blkdeactivate`

### PR DESCRIPTION
**Pull request by jfroche**
Zap script uses `blkdeactivate` to deactivates block devices. `blkdeactivate` is currently in the lvm2 package.
Add grep/find/cut tools as well as we are fixing complete PATH.

### CODE REVIEW

File: types.nix [hunk 1]
```diff
@@ -606,7 +606,7 @@ rec {
         readOnly = true;
         type = types.functionTo (types.listOf types.package);
         default = pkgs:
-          [ pkgs.parted pkgs.systemdMinimal ] ++ flatten (map (partition: partition._pkgs pkgs) config.partitions);
+          [ pkgs.parted pkgs.systemdMinimal pkgs.lvm2 pkgs.gnugrep pkgs.coreutils pkgs.findutils ] ++ flatten (map (partition: partition._pkgs pkgs) config.partitions);
```
Comment (Mic92) [inline types.nix:609]: Could be lvm2 moved to lvm partitions?

Comment (jfroche) [reply types.nix:609]: We need `blkdeactivate` from lvm2 package even if we don't use lvm

Comment (Mic92) [reply types.nix:609]: I see. Maybe add a comment why this is needed.

**Comment by Mic92** [comment]
Will this be deprecated by https://github.com/nix-community/disko/pull/75 ?

**Pull request closed without merging**